src/main/perl/LV.pm

```diff
@@ -59,7 +67,19 @@ sub _initialize
     $self->{volume_group} = NCM::LVM->new(BASEPATH . VGS . $st->{volume_group}, $config);
     $self->{size}         = $st->{size};
     $self->{stripe_size}  = $st->{stripe_size} if exists $st->{stripe_size};
-    
+    if (exists $st->{cache}) {
```

Comment:
what are the possible values of `$st->{cache}`? (is the exists really required?)